src/main/java/org/opensearch/security/rest/SecurityInfoAction.java

```diff
@@ -122,7 +122,7 @@ public void accept(RestChannel channel) throws Exception {
                     final User user = threadContext.getTransient(ConfigConstants.OPENDISTRO_SECURITY_USER);
                     final TransportAddress remoteAddress = threadContext.getTransient(ConfigConstants.OPENDISTRO_SECURITY_REMOTE_ADDRESS);
 
-                    final Set<String> securityRoles = evaluator.mapRoles(user, remoteAddress);
+                    PrivilegesEvaluationContext context = evaluator.createContext(user, null);
```

Comment:
qq, is remoteAddress not being used when getting mappedRoles?

Reply:
The second parameter is the action name for `createContext()`, not the `remoteAddress`. `createContext()` will retrieve the `remoteAddress` by itself out of the thread context (i.e., do that what is also being done in line 123 as seen above).

We can pass `null` for the action name, as we are not actually authorizing any action here.